lib/route.js

```diff
@@ -148,11 +148,11 @@ function buildRouting (options) {
   }
 
   function hasRoute ({ options }) {
-    return router.find(
+    return router.hasRoute(
       options.method,
       options.url || '',
       options.constraints
-    ) !== null
+    )
```

Comment:
The return type here should stay a boolean

```suggestion
    ) !== null
```

Reply:
Already returns a `boolean` isn't it?
https://github.com/delvedor/find-my-way?tab=readme-ov-file#hasroute-method-path-constraints

Reply:
Yeah, just woke up 😁🤣